### ISSUE TITLE
Antd 4.x Tree component has Accessibility issue using Axe Accessibility tool: "Form elements must have labels". Why an <input> as part of Tree and how can I label it

### ISSUE THREAD

**Issue by abhishekahuja87** [ant-design/ant-design#31216]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/embed/customize-collapseexpand-icon-antd4166-f3my8?fontsize=14&hidenavigation=1&theme=dark)

### Steps to reproduce
- Install Axe Accessibility Tool on Chrome browser
- Run the tool against the app in codesandbox link, find the accessibility warnings

### What is expected?
- No Serious accessibility warnings

### What is actually happening?
- Serious warning present

| Environment | Info |
|---|---|
| antd | 4.16.6 |
| React | Latest |
| System | Any |
| Browser | Any |

**Comment by afc163** [comment]
Trace #31012

**Comment by zersiax** [comment]
Axe is right in this case, but only sees the bare minimum of what is actually wrong with this control. Essentially this entire treeview control is an accesibility nightmare at present:

- Not marked up as a semantic treeview
- - Items within the treeview are not marked up as treeviewItem
- - Screenreaders can't efficiently navigate the treeview
- - If keyboard support works at all, I can't verify this given the screenreader isn't notified when focus changes from one node to another.

So yeah, Axe is right, there shouldn't be an input control in there and that it's missing a label is an accessibility issue, but overall this entire component is currently broken for any kind of assistive tech.
